.ko.yaml

```diff
@@ -7,3 +7,6 @@ defaultPlatforms:
 defaultLdflags:
   - -s
   - -w
+  - -X github.com/kro-run/kro/pkg.Version={{.Env.RELEASE_VERSION}}
+  - -X github.com/kro-run/kro/pkg.GitCommit={{.Env.GIT_COMMIT}}
```

Comment:
I assume this automagic is handled by KO ?

Reply:
Correct https://ko.build/configuration/#overriding-go-build-settings